metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -818,6 +820,17 @@ def gen_dataset_workunits(
             project_id, dataset_name, table.name
         )
 
+        # Added for bigquery to gcs lineage extraction
+        if (
+            isinstance(table, BigqueryTable)
+            and table.table_type is not None
+            and table.table_type == "EXTERNAL"
+            and table.ddl is not None
+            and f"CREATE EXTERNAL TABLE `{project_id}.{dataset_name}.{table.name}`"
```

Comment:
Is this expression always uppercase, and is there always one space between words?

Reply:
If this can happen the you need a case insensitive regexp match like this:
`.*create\s+external\s+table\s+[`]?(project_id\.)?.*[`]?`

Here are some textcase if this stands:
```
CREATE EXTERNAL TABLE `project_id.dataset_name.table_name`
CREATE EXTERNAL TABLE `dataset_name.table_name`
CREATE EXTERNAL TABLE dataset.table_name
CREATE EXTERNAL TABLE project_id.dataset_name.table_name
   create    external  table   `project_id.dataset_name.table_name`
   create    external  table   project_id.dataset_name.table_name
```

Reply:
Yes this expression always uppercase, i ll make that change